### PR TITLE
Adding Game Files Article

### PR DESCRIPTION
**Pull request by PavlovTM**
This PR adds an article which explains the game files with examples, pictures including useful side information.

Perhaps over the top, but very useful to explain modding to newcomers

### CODE REVIEW

File: articles/references/game-files.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+|   O    | ps4 / orbis                   |
+|   A    | android                       |
+|   S    | ios                           |
+|   P    | ps5 / prospero                |
+|   N    | nx64                          |
+|   X    | xbox360 / xenon               |
+|   C    | ps3 / psn                     |
+|   I    | independent (North-specific)  |
+
+### Suffix
```
Comment (C0kkie) [inline articles/references/game-files.md:23]: The Links in the suffex topics doesnt work

File: articles/references/game-files.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+   Editing based on Polygons is only available in 3ds Max & ZModeler3.
+
+## YND
+[comment]: <> "Path Node"
+
+### Explanation
+It declares Path Nodes required for NPCs and calculating GPS Routes.
+
+### Example
+This is an example inside CodeWalker, how Path Nodes look like
```
Comment (C0kkie) [inline articles/references/game-files.md:79]: Add a line break at the end of line

File: articles/references/game-files.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+## YMAP
+[comment]: <> "Map Data"
+
+### Explanation
+Collection of Entities to form a Compound.
+You can create it inside CodeWalker, together with a [YMF (Manifest File)](##YMF)
+
+### Example
+This is the YMAP, that gets included into your CodeWalker Project.
+It contains several Entities,
```
Comment (C0kkie) [inline articles/references/game-files.md:113]: Add a line break at the end of line

File: articles/references/game-files.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+### Explanation
+Mainly used for Vehicles, usually contains an High Details alternative.
+
+| Low Details | High Details |
+| :---------: | ------------ |
+|   name.yft  |  hi@name.yft |
+
+### Example
+Inside ZModeler, a vehicle in hi@ format looks like this
```
Comment (C0kkie) [inline articles/references/game-files.md:137]: Add a line break at the end of line

File: articles/references/game-files.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+<img src="https://i.imgur.com/IeC1hTO.png" alt="YFT Example" style="zoom: 75%"/>
+
+## YDD
+[comment]: <> "Drawable Dictionary"
+
+### Explanation
+Used for complex Buildings, usually contains several LODs and Models.
+
+### Example
+Inside OpenIV, it looks like this
```
Comment (C0kkie) [inline articles/references/game-files.md:147]: Add a line break at the end of line

File: articles/references/game-files.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+## YDR
+[comment]: <> "Drawables"
+
+### Explanation
+Small and simple objects, containing embedded textures or external Textures via. [YTD](##YTD)
+Used for Entities, which are combined into a [YTYP](##YTYP)
+
+### Example
+Inside OpenIV, an YDR looks like this.
```
Comment (C0kkie) [inline articles/references/game-files.md:158]: Add a line break at the end of line

**Review by C0kkie — CHANGES_REQUESTED** [review]

File: articles/references/game-files.md (hunk 1 above)
Comment (PavlovTM) [reply articles/references/game-files.md:23]: Is there something special about anchor links on the docs site?
It's something so simple

File: articles/references/game-files.md (hunk 6 above)
Comment (PavlovTM) [reply articles/references/game-files.md]: Resolved

File: articles/references/game-files.md (hunk 5 above)
Comment (PavlovTM) [reply articles/references/game-files.md]: Resolved

File: articles/references/game-files.md (hunk 4 above)
Comment (PavlovTM) [reply articles/references/game-files.md]: Resolved

File: articles/references/game-files.md (hunk 3 above)
Comment (PavlovTM) [reply articles/references/game-files.md]: Resolved

File: articles/references/game-files.md (hunk 2 above)
Comment (PavlovTM) [reply articles/references/game-files.md]: Resolved

File: articles/references/game-files.md (hunk 1 above)
Comment (C0kkie) [reply articles/references/game-files.md:23]: Use singlehash and lowercase and it should work

**Comment by PavlovTM** [comment]
Changed, I'd say

File: articles/references/game-files.md (hunk 1 above)
Comment (Unnvaldr) [reply articles/references/game-files.md:23]: Will be addressed outside this pR

**Review by Unnvaldr — APPROVED** [review]
Thanks, seems fine now

**Pull request merged**